app/src/components/EditEvent/EditEventContainer.tsx

```diff
@@ -68,6 +77,13 @@ export const EditEventContainer = () => {
     history.push(eventsRoute);
   });
 
+  const isDisabled = !isOk(event);
+  const validatePreview = () => {
+    if (isOk(event)) {
+      setPreviewState(true);
+    }
+  };
```

Comment:
Synes det er litt rart at `validatePreview` setter en verdi i tillegg til å validere. Men jeg kommer ikke på noe bedre navn for den selv 🙈